tests/xtimer_usleep_short/tests/01-run.py

```diff
@@ -12,19 +12,27 @@
 
 
 def testfunc(child):
-    child.expect(u"This test will call xtimer_usleep for values from \\d+ down to \\d+\r\n")
+    child.expect(u"This test will call xtimer_usleep for values from (\d+) down to (\d+)")
```

Comment:
Now that we are using python3 none of the string required the `u` in front anymore.